### PR TITLE
Github Actions ci/cd config to test dependencies & src code automatically, as well as to deploy to npm registry

### PR DESCRIPTION
**Pull request by nemanja255**
This merge contains config changes for ci/cd

### CODE REVIEW

File: .github/workflows/ci-cd.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          path: /tmp/nblocks-ts-client.tar
+  test:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout Github Repository
+      - name: Step 1 - Checkout Github Repository
+        uses: actions/checkout@v3
+      # Add parallel builds for multi platform support
+      - name: Step 2 - Set up QEMU
```
Comment (orr3) [inline .github/workflows/ci-cd.yml:48]: Is Qemu and Buildx (Step 2 & Step 3) really necessary even though we just pull an existing image and run it?

File: .github/workflows/ci-cd.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  deploy:
+    needs: [build, test]
+    if: ${{ github.ref_name == 'master' && github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout Github Repository
+      - name: Step 1 - Checkout Github Repository
+        uses: actions/checkout@v3
+        # Add parallel builds for multi platform support
+      - name: Step 2 - Set up QEMU
```
Comment (orr3) [inline .github/workflows/ci-cd.yml:73]: Is Qemu and Buildx (Step 2 & Step 3) really necessary even though we just pull an existing image and run it?

**Review by orr3 — CHANGES_REQUESTED** [review]
Everything looks great. Just had one question

File: .github/workflows/ci-cd.yml (hunk 1 above)
Comment (nemanja255) [reply .github/workflows/ci-cd.yml:48]: I don't think we need it. When we talked about it I understood that I should leave it for later, testing the build on other arch...

Comment (nemanja255) [reply .github/workflows/ci-cd.yml:48]: Would you like me to remove them for this merge?

Comment (orr3) [reply .github/workflows/ci-cd.yml:48]: Yes, if they are not needed it’s clearer if they’re removed or at least commented out (with a comment describing why)

**Pull request merged**